### PR TITLE
Fix minor lint issues

### PR DESCRIPTION
**Pull request by nopcoder**

### CODE REVIEW

File: cloud/aws/s3inventory/reader_test.go [hunk 1]
```diff
@@ -72,6 +72,9 @@ func generateParquet(t *testing.T, objs <-chan *TestObject, fieldToRemove string
 		_ = fw.Close()
 	}()
 	jsonSchema, err := json.Marshal(parquetSchema(fieldToRemove))
+	if err != nil {
+		t.Fatalf("failed to json marshal: %s", err)
```
Comment (arielshaqed) [inline cloud/aws/s3inventory/reader_test.go:76]:
```suggestion
		t.Fatalf("failed to marshal to JSON %+v: %s", parquetSchema(fieldToRemove), err)
```

File: export/tasks_generator_test.go [hunk 2]
```diff
@@ -153,15 +153,6 @@ func makeAllDependencies(dd directDependencies) allDependencies {
 	return &allDependenciesSet{dd: dd, s: make(map[parade.TaskID]map[parade.TaskID]struct{})}
 }
 
-func validateAllAfter(t *testing.T, d allDependencies, typ string, before parade.TaskID, afters []parade.TaskID) {
```
Comment (arielshaqed) [inline export/tasks_generator_test.go:156]: This one's mine.  Note to self: verify that we don't really need to use it any longer.

File: graveler/ref/resolver_test.go [hunk 3]
```diff
@@ -43,10 +43,11 @@ func TestRefManager_Dereference(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 
-	ids := make([]graveler.CommitID, 0)
 	for iter.Next() {
 		c := iter.Value()
-		ids = append(ids, c.CommitID)
```
Comment (arielshaqed) [inline graveler/ref/resolver_test.go:49]: I don't understand.  Where do we validate that we actually received good values, or even the correct number of values  E.g. if the iterator never iterates and ends directly, this passes.  (Not introduced in this PR, of course, so fine to open an issue and assign to whoever should fix)

File: parade/parade_test.go [hunk 4]
```diff
@@ -22,16 +22,10 @@ import (
...
-const (
-	dbContainerTimeoutSeconds = 10 * 60 // 10 min
-	dbName                    = "parade_db"
-)
-
 var (
 	pool        *dockertest.Pool
 	databaseURI string
 
-	postgresUrl = flag.String("postgres-url", "", "Postgres connection string.  If unset, run a Postgres in a Docker container.")
```
Comment (arielshaqed) [inline parade/parade_test.go:34]: Yikes!  Good catch...

**Review by arielshaqed — APPROVED** [review]
Cool!

What changed?  Why are these (finally...) failing to lint?

**Comment by nopcoder** [comment]
> Cool!
> 
> What changed?  Why are these (finally...) failing to lint?

didn't fail, we are just graceful with the tests code.

File: export/tasks_generator_test.go (hunk 2 above)
Comment (arielshaqed) [reply export/tasks_generator_test.go:156]: Reply to self: D'uh, you removed it in favour of a better checked with full transitivity that lets you test dependencies directly, e.g. `expectedDeps` on l.358.

**Comment by nopcoder** [comment]
the build fails after adding err check - so I still need to verify if ignoring it was a bug or there is a bug

File: catalog/mvcc/cataloger_delete_entry_test.go [hunk 5]
```diff
@@ -150,8 +151,11 @@ func TestCataloger_DeleteEntryAndCheckItRemainsInCommits(t *testing.T) {
 	if err != nil {
 		t.Fatal("Failed to commit after delete:", err)
 	}
-	entry, err = c.GetEntry(ctx, repository, nextCommit.Reference, "/file2", catalog.GetEntryParams{})
-	entry, err = c.GetEntry(ctx, repository, prevCommit.Reference, "/file2", catalog.GetEntryParams{})
+	_, err = c.GetEntry(ctx, repository, nextCommit.Reference, "/file2", catalog.GetEntryParams{})
+	testutil.Must(t, err)
```
Comment (arielshaqed) [inline catalog/mvcc/cataloger_delete_entry_test.go:155]: Shouldn't the file be missing here?  Indeed, the test appears to fail.

**Review by arielshaqed — CHANGES_REQUESTED** [review]
Requesting changes because test is failing

File: graveler/ref/resolver_test.go (hunk 3 above)
Comment (nopcoder) [reply graveler/ref/resolver_test.go:49]: Adding a test code to compare the values.
I think it was made just to check there was no error after scanning the commit log.

File: catalog/mvcc/cataloger_delete_entry_test.go (hunk 5 above)
Comment (nopcoder) [reply catalog/mvcc/cataloger_delete_entry_test.go]: Will verify that the file is missing

File: graveler/ref/resolver_test.go [hunk 6]
```diff
@@ -43,37 +44,44 @@ func TestRefManager_Dereference(t *testing.T) {
...
+		"bfa1e0382ff3c51905dc62ced0a67588b5219c1bba71a517ae7e7857f0c26afe",
+		"d2248dcc1a4de004e10e3bc6b820655e649b8d986d983b60ec98a357a0df194b",
+		"a2d98d820f6ff3f221223dbe6a22548f78549830d3b19286b101f13a0ee34085",
+		"4f13621ec00d4e44e8a0f0ad340224f9d51db9b6518ee7bef17f598aea9e0431",
+		"df87d5329f4438662d6ecb9b90ee17c0bdc9a78a884acc93c0c4fe9f0f79d059",
+		"29706d36de7219e0796c31b278f87201ef835e8cdafbcc3c907d292cd31f77d5",
+	}
+
+	if diff := deep.Equal(commitIDs, commitLog); diff != nil {
+		t.Fatal("Difference found on commit log", diff)
```
Comment (arielshaqed) [inline graveler/ref/resolver_test.go:83]:
```suggestion
		t.Error("Difference found on commit log", diff)
```
(the test can continue regardless, and the additional error messages may help understand what went wrong -- e.g. if `commitIDs` is empty.)

**Review by arielshaqed — APPROVED** [review]
Nice, thanks!

**Comment by nopcoder** [comment]
> Nice, thanks!

Thank you.

**Pull request merged**